app/Controllers/authController.php

```diff
@@ -136,7 +128,7 @@ public function formLoginAction(): void {
 				return;
 			}
 
-			usleep(random_int(100, 10000));	//Primitive mitigation of timing attacks, in μs
+			usleep(rand(100, 10000));	//Primitive mitigation of timing attacks, in μs
```

Comment:
Could you please keep this PR clean of unrelated modifications like changing the random functions?
If any is worth doing, let's address it in another PR